protocol/messenger_contact_requests_test.go

```diff
@@ -1472,7 +1560,7 @@ func (s *MessengerContactRequestSuite) blockContactAndSync(alice1 *Messenger, al
 	s.Require().Equal(ContactRequestStateReceived, respContact.ContactRequestRemoteState)
 
 	// Check chats list
-	//s.Require().Len(alice2.Chats(), 1) // FIXME: Uncomment after https://github.com/status-im/status-go/issues/3800
+	s.Require().Len(alice2.Chats(), 2)
```

Comment:
@igor-sirotin, maybe it's time to remove https://github.com/status-im/status-go/blob/develop/protocol/messenger_contacts.go#L414-L415 ?

Reply:
I think not yet. Maybe in a year or smth 😄

Reply:
But it's better to use `AddChatsCount` in such cases.

https://github.com/status-im/status-go/blob/d535cd95f885a30387e0333f71c9a1081fc72a3a/deprecation/protocol.go#L8-L17